src/EventGenerator/EmitEvent.php

```diff
@@ -132,6 +145,11 @@ public static function create(ContainerInterface $container, array $configuratio
   public function execute($entity = NULL) {
     // Generate event as stomp message.
     try {
+      if (is_null($this->stomp->getClient()->getProtocol())) {
```

Comment:
I see what you're saying. I was (maybe naively) hoping that if the JMS service was still up this might restore your connection. But based on what I understand to be the lifecycle of the client, it is probably unlikely to bounce in a single request. So I can just stick the logging/messenger stuff in here and `return;`

Reply:
I don't disagree with the retry attempt, just was confirming the expectations was all!

Reply:
I looked at how the to have the stomp service kill itself, but it was not immediately apparent and so my mind went to a new service to wrap the StatefulStomp client. That seemed like overkill. I'll add a comments requested at the disconnect + connect.